.github/workflows/pytest.yml

```diff
@@ -0,0 +1,17 @@
+name: pytest
+
+on: [push, pull_request]
+
+jobs:
+  python-pytest:
+    name: Python pytest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Install python dependencies
+        run: |
+          python -m pip install --upgrade pip virtualenv
+          python -m pip install --upgrade pytest
```

Comment:
We do not use `virtualenv`, do we?
And `python3-pytest` is in debian. Couldn't we just `sudo apt install python3-pytest`?
(I guess the same comment could be made if we did `apt install` here. "Could'nt we just pip install instead?". Maybe it just comes down to personal preference)

Reply:
Yes, virtualenv is not used. I'd prefer to install from pypi, as it's easier to get new versions that way.